Added another when condition in mount nfs share and removed wrong item.1.stdout

Rendanic opened the pull request:
I tried an installation with extracted archives on a local machine with following variables:
install_from_nfs:     False
oracle_sw_copy:       False
oracle_sw_unpack:     False

This is not possible, because the Task 'Mount nfs share with installation media' is executed and not required in this situation.

I removed the item.1.stdout != "1"  due to strange errors in Ansible while running the playbook. They were removed at other steps and I think we don't need them.

oravirt commented:
Hi,
What exactly were the strange errors you got with the item.1.stdout condition?

Rendanic commented:
Hi,
I realized that the item.1.stdout is a failure by myself. I added a lot of tags to 'Check if DB-server is already installed' to fill the variable checkdbswinstall for the other tasks in oraswdb-install.
You could reject the commit for 'removed item.1.stdout in role oraswdb-install'.

I am currently working on a lot of new feautures for oracle-ansible.
Added variables in oracle_databases:
- oracle_home
- characterset
- nationalcharacterset
Added variables in host_fs_layout:
- lvsize

All changes are done without breaking the current functions of oracle-ansible.
There is currently some more work before I could create the pull request for all the additions.

Thanks for the great work!

Rendanic commented:
I replaced this pull request with a new one.
